### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Use a SAS URL_Attach with SAS URL_ScreenReader: Voiceover does not read the title of the dialog when select the Use a shared access signature (SAS) URI radio button and next. 

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#1975]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1;#MAS2.4.2 #VoiceOver; #Mac; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
MAC Version 10.14.6 Beta(18G29g)
**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Use a SAS URL" radio button and hit enter on Next button.
4. "Attach with SAS URL" screen gets opened.
5. Start VoiceOver and using Tab key and check whether the Voiceover reads the title of the dialog when opened.

**​Actual:​**
Voiceover does not read the title of the dialog when select the Use a shared access signature (SAS) URI radio button and next. 

Same issue is observed for the below screen.
Attach with Azure AD 
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Add a resource via Azure Active Directory(Azure AD)" radio button and hit enter on Next button.
4. "Attach with Azure AD" screen gets opened.

**Expected:**
VoiceOver should read the title as Microsoft Azure Storage Explorer Connector Attach with SAS Url dialog .

**UserImpact:**
Screen reader users unable to know the which dialog opened if it will not read the title.

Recommendation:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
MAS1.3.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
MAS2.4.2:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={c74ff881-82b6-4401-92cf-8919f9bffaf2}
**Attachment for Reference:**
[1975_A11y_AzureToolsStorageExplorer_Use a SAS URL_Attach with SAS URL_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3669174/1975_A11y_AzureToolsStorageExplorer_Use.a.SAS.URL_Attach.with.SAS.URL_ScreenReader.pptx)

**Comment by Suraj1200499** [comment]
The above logged issue is observed for all popup present in entire application.

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo and @Suraj1200499 I'm confused by this issue. When I use VoiceOver (see attached), there are no problems with speaking that the current window is the connect dialog. As you can see in my recording, VoiceOver says: "Microsoft Azure Storage Explorer, Microsoft Azure Storage Explorer - Connect, window, URI:, edit text".
![1975](https://user-images.githubusercontent.com/21112697/69370343-a5fe9d00-0c52-11ea-8b7e-07fb1fa98f9f.gif)

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo and @Suraj1200499 can you please follow up?

**Comment by Suraj1200499** [comment]
@MRayermannMSFT 
When Screen gets Opened VoiceOver should say the exact title of the dialog.
For eg. possibly, VoiceOver should announce, "Microsoft Azure Storage Explorer - Connect, Attach with SAS URL window".
Same issue is Observed in the entire application for all the dialogs.

**Comment by MRayermannMSFT** [comment]
@suraj1200499 , that is not the title of the dialog. The title is "Connect". Which in my gif you can see VoiceOver speaking. Attach with SAS URL is the title of the current page.

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo and @Suraj1200499 can you please follow up?

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo and @Suraj1200499 can you please follow up?

**Comment by nirmalsahoo** [comment]
We have verified the issue and is still reproduced. Voiceover should read the title as Attach with SAS URI.This issue is reproduced throughout the all dialog of the application.

**Comment by MRayermannMSFT** [comment]
I understand that you still see the behavior, what I'm trying to say is that the behavior is by design. The title of the dialog is "Connect". The title of the page is "Attach with SAS URI" That information is available to all users with the page title being printed on the page.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT 
The bug is By design, As per our process By design bug will be validated by our SME's. So I had put this bug in their note. Once I get a reply from them, I will update the comment.

**Comment by MRayermannMSFT** [comment]
@nirmalsahoo, I went ahead and joined office hours this week with Ryan. He agrees that the title of each page does not need to be added to the title of the dialog itself. Also, fyi, for the other 2 bugs you and I have been disagreeing on (#2069 and #2093), I'm going to take one final look at those.

**Comment by MRayermannMSFT** [comment]
Closing as by design.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
